include/dlpack/dlpack.h

```diff
@@ -222,6 +222,12 @@ typedef struct DLManagedTensor {
    *   The destructors deletes the argument self as well.
    */
   void (*deleter)(struct DLManagedTensor * self);
+  /* The version of the DLManagedTensor */
+  int32_t version;
```

Comment:
Instead of introducing a new struct with a DLPack and ABI version, we can just export the DLPack version since it uniquely determines the ABI version. (cc @mattip)